db/table_cache.h

```diff
@@ -170,6 +171,11 @@ class TableCache {
     }
   }
 
+  uint64_t ApproximateSize(const FdWithKeyRange& f, const Slice& key,
```

Comment:
Need to add comment about what the function does. Also, does it makes sense to call it `ApproximateOffsetOf()` instead, considering that we are returning the file offset up to the key?